src/aspire/denoising/class_avg.py

```diff
@@ -459,7 +464,7 @@ class LegacyClassAvgSource(ClassAvgSource):
     Defaults to using global variance based class selection,
     and a brute force image alignment (rotational only).
```

Comment:
Does this need to be updated? The description gives the impression we are using brute force rotations.

Reply:
Hrmm, this is a bit of personal choice in definitions. I changed it in a way that I hope avoids disagreements.

Reply:
Ok, sounds good to me!